src/hooks/FilterQuestionType.ts

```diff
@@ -0,0 +1,21 @@
+export const filterQuestionType = (questionType: string) => {
+  let output = '';
+  switch (questionType) {
+    case 'A':
+      output = '협업';
+      break;
+    case 'B':
+      output = '동기부여';
+      break;
+    case 'C':
+      output = '성장';
+      break;
+    case 'D':
+      output = '개인생활';
+      break;
+    case 'E':
+      output = '건강';
+      break;
+  }
+  return output;
```

Comment:
```suggestion
export const filterQuestionType = (questionType: string) => {
  switch (questionType) {
    case 'A':
      return '협업';
    case 'B':
      return '동기부여';
    case 'C':
      return '성장';
    case 'D':
      return '개인생활';
    case 'E':
      return '건강';
  }
};
```
이렇게 작성하면 코드를 줄일 수 있습니당!